SCons/Action.xml

```diff
@@ -200,4 +200,26 @@ in which the command should be executed.
 </summary>
 </cvar>
 
+<cvar name="SHELL_ENV_EXPANDER">
+  <summary>
+    <para>
+A function to obtain the environment dictionary which will be used
+when executing the &cv-link-SPAWN; function. This primarily give the
+user a chance to customize the shell environment for particular Actions.
```

Comment:
I've been using the specific term "execution environment" for this in recent doc updates - perhaps think if that would make sense here too?

Reply:
updated per suggestion in the latest